src/dnn/MIOpen.jl

```diff
@@ -0,0 +1,77 @@
+module MIOpen
+
+import AMDGPU
+import AMDGPU.Runtime.Mem
+
+using AMDGPU: ROCArray, ROCDevice
```

Comment:
```suggestion
import .AMDGPU: ROCArray, ROCDevice
```

I think this is equivalent? I like to avoid `using` unless we need it to get exported things.